docs/checks/invalidTagsCheck.md

```diff
@@ -1,17 +1,17 @@
 # Invalid Tags Check
 
 This check flags features for invalid tags based on a list of TaggableFilters and RegexTaggableFilters. The list of filters for this check are passed through the config and/or through resource files.
-The two configurable boolean values, "filters.resource.append" and "filters.resource.override", 
+The two configurable boolean values, "filters.resource.append" and "filters.resource.override",
```

Comment:
Are these changes to resolve some sonar issues?

Reply:
No. My text editor (vim) is set up to autotrim trailing spaces on lines, which doesn't always make sense (specifically, for markdown files).